src/Components/login/Login.js

```diff
@@ -18,10 +18,18 @@ const Login = () => {
   const [emailText, setEmailText] = useState("");
   const [passwordText, setPasswordText] = useState("");
   const [showPassword, setShowPassword] = useState(false);
-  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [isLoggedInHook, setIsLoggedInHook] = useState(false);
```

Comment:
no need to rename these two, leave them as they were before

Reply:
ok, I already renamed them